internal/pkg/stream/cloudformation.go

```diff
@@ -124,9 +124,7 @@ func (s *StackStreamer) Notify() {
 	// notifying previous subscribers of older events.
 	s.mu.Lock()
 	var subs []chan StackEvent
-	for _, sub := range s.subscribers {
-		subs = append(subs, sub)
-	}
+	subs = append(subs, s.subscribers...)
```

Comment:
Ah, clever!

Reply:
staticcheck yelled at me to make this change :)